fccodegen/_generator.py

```diff
@@ -38,21 +67,37 @@ def load_model(path):
     return model
 
 
-def get_equations(graph):
+def get_equations_for(graph, symbols):
     """
-    Gets the equations from a cellml model graph.
+    Gets the equations from a cellmlmanip model to calculate the values of the
+    given symbols.
     """
-    #TODO: This should become part of cellmlmanip
+    #TODO: This should become part of cellmlmanip, with a syntax like
+    # get_equations_for(variables), and then called with the derivatives to
+    # generate the RHS, or with the outputs to generate the output code.
 
     # Get sorted symbols
-    #TODO: networkx is not a dependency
+    #TODO: networkx is not a dependency of fccodegen - only of cellmlmanip. If
+    #      this does _not_ get moved to cellmlmanip, it should become a dep.
+    #      here too.
     import networkx as nx
-    symbols = nx.lexicographical_topological_sort(graph, key=lambda x: str(x))
+    sorted_symbols = nx.lexicographical_topological_sort(
+        graph, key=lambda x: str(x))
+
+    # Create set of symbols for which we require equations
```

Comment:
It'll be more efficient (particularly for outputs) if you can do a topological sort just on the sub-graph that's actually required. But save implementing this for `cellmlmanip`.

Reply:
Yeah it's a silly implementation! But I just wanted something that'd work with the 5 API calls I knew about :-)